app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/ProjectToProjectLayerMiddleware.scala

```diff
@@ -14,7 +11,15 @@ import doobie.implicits._
 import org.http4s._
 import org.http4s.dsl.io._
 
+import java.util.UUID
+
 object ProjectToProjectLayerMiddleware {
+  private def getDefaultLayerId(projectId: UUID,
```

Comment:
Seems to me that this would be more useful in ProjectDao

Reply:
The only reason it's not is the different signature -- we almost exclusively return `ConnectionIO`s in the `FooDao` methods (except for the two or three that return `Stream[IO, T]`s), so I did this here just to avoid the `OptionT { ... }` wrapper in three places. Given that we don't always know we'll want an `OptionT[IO, UUID]` when we get the default layer id, I thought it'd make more sense to keep the tiny helper for that here